pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -2454,8 +2454,8 @@ func TestEval(t *testing.T) {
 				}
 			}
 			`,
-			expectedStatusCode: http.StatusUnauthorized,
-			expectedMessage:    "user is not authorized to query one or many data sources used by the rule",
+			expectedStatusCode: http.StatusBadRequest,
+			expectedMessage:    "invalid condition: invalid query A: data source not found: unknown",
```

Comment:
question: are these test changes related to the other changes or a separate fix?